config/routes.rb

```diff
@@ -12,6 +12,16 @@
     member do
       get :assign
       patch :assign
+
+      resources :policy_assessments do
+        get :part, on: :new
+
+        collection do
+          get "/:part/:policy_class", action: "show_class"
+          put "/:part/:policy_class", action: "update_class"
```

Comment:
These routes feel wrong - why are they on the policy assessment collection? Don't they apply to specific policy assessments?